### DISCUSSION TITLE
'problem too large' when merging ATAC objects

### DISCUSSION THREAD

**Discussion by dru93** [stuart-lab/signac#1276] [Q&A]
I'm trying to merge 43 atac samples via the merge command as in the "Merging objects" vignette. After a point this error:
`Error in .cbind2Csp(x, y) :
  Cholmod error 'problem too large' at file ../Core/cholmod_sparse.c, line 89`
I'm not really sure what exactly does that mean, but I assume has to do with the fact that the objects I'm trying to merge are pretty huge, but is there any way to bypass this error?
Thanks!

**Comment by timoast** [comment]
How many cells and peaks do you have?

**Reply by dru93** [reply]
I have 15839260 cells and 391451 peaks split between the two objects I'm trying to combine

**Reply by timoast** [reply]
It's simply too large for the current Seurat/Signac infrastructure. We use the sparse matrix format from the Matrix package, which is limited to 2^32-1 non-zero elements, which your dataset would exceed. We're working on improvements to overcome this limitation but don't have anything available right now.

All I can suggest is keeping the data in smaller objects, if that's possible for your analysis
